### ISSUE TITLE
Support forked processes

### ISSUE THREAD

**Issue by lwerdna** [Vector35/debugger#84]
conversation from lexsek:

Just a TAB to send GDB commands (could be usefull for forks -> follow-fork-mode [child|parent] and so on
Or just the ability to choose between the processes created by forks
That sould be also great to follow spawned processes by the main debugged process
2:30
Maybe a "Process TAB" just like the thread one
New
2:32
But yeah basically beeing able to choose easily which process to follow (inside binja impletemented stuff, or just a tab to send gdb commands to start like follow-fork-mode) would be awesome

**Comment by CouleeApps** [comment]
Does gdb support targeting multiple processes at the same time? Or would we need to maintain a new instance of gdb (and another adapter) for every child process?

**Comment by themrrobert** [comment]
I believe this answers your question @CouleeApps 
https://sourceware.org/gdb/onlinedocs/gdb/Forks.html
tl;dr; System dependent. Linux Kernel 2.5.46+ has support for this which gdb utilizes,

Total guess/personal theory:
WinDBG supports this with .chliddbg 1, so there may be [un]documented support in the windows kernel which gdb could be made to tap into.
